doc/TRAINING.rst

```diff
@@ -124,7 +124,7 @@ It will also add the following ``.csv`` files:
 * ``clips/dev.csv``
 * ``clips/test.csv``
 
-All entries in these CSV files refer to their samples by absolute paths. So moving this sub-directory would require another import or tweaking the CSV files accordingly.
+All entries in these CSV files refer to their samples by relative paths.
```

Comment:
@karansag Could you change the wording to reflect that it can be relative or absolute?